roles/ceph-common/defaults/main.yml

```diff
@@ -360,3 +360,5 @@ dmcrypt_journal_collocation: False
 dmcrypt_dedicated_journal: False
 
 osd_auto_discovery: False
+
```

Comment:
Can you add a comment to explain what the variable does? Then run: `generate_group_vars_sample.sh`.